providers/stores/file.js

```diff
@@ -95,14 +91,19 @@ class FileStore extends AbstractStore {
     // a) all fit in memory reasonably, and
     // b) fit in one list call (i.e., <5000)
     const files = await recursive(root);
-    const contents = await Promise.all(files.map(file => {
-      return new Promise((resolve, reject) =>
-        fs.readFile(file, (error, data) =>
-          error ? reject(error) : resolve({ name: file, content: JSON.parse(data) })));
-    }));
+    const contents = await Promise.all(
+      files.map(file => {
+        return new Promise((resolve, reject) =>
+          fs.readFile(
+            file,
+            (error, data) => (error ? reject(error) : resolve({ name: file, content: JSON.parse(data) }))
```

Comment:
not very happy about this but it could handled by declaring the callback function before the `readFile` and then just passing that in. changes what looks like a scary nested structure into simple, declare callback, pass callback.